pkg/cmd/root/root.go

```diff
@@ -29,6 +30,9 @@ func NewRootCommand(f *factory.Factory, version string) *cobra.Command {
 		Short:         f.Localizer.MustLocalize("root.cmd.shortDescription"),
 		Long:          f.Localizer.MustLocalize("root.cmd.longDescription"),
 		Example:       f.Localizer.MustLocalize("root.cmd.example"),
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			f.Logger.SetDebug(debug.Enabled())
```

Comment:
That will run for some meta commands like __completion

Reply:
I would set this as part of the run code

Reply:
> I would set this as part of the run code

Do you recommend any particular place for that? I can check if debug is reported as enabled there.

Reply:
After line 39 in the same file seems more cleaner

Reply:
After arguments.AddDebugFlag(fs)

Reply:
> After arguments.AddDebugFlag(fs)

Unfortunately debug.Enabled() returns false there...

![image](https://user-images.githubusercontent.com/499857/132992066-9ed732ba-7c51-4710-ac3e-1e2d4712448e.png)